modules/syslog-configuring-acs.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-using-syslog-protocol.adoc
+:_module-type: PROCEDURE
+[id="syslog-configuring-acs_{context}"]
+= Configuring {product-title}
```

Comment:
I mentioned it in another PR for a similar module, but perhaps the title of this should be more specific to what you're configuring (a syslog integration with RHACS).